lib/main_common.pm

```diff
@@ -1514,7 +1508,10 @@ sub load_extra_tests_console {
     loadtest "console/wget_ipv6";
     loadtest "console/ca_certificates_mozilla";
     loadtest "console/unzip";
-    loadtest "console/salt" if is_jeos;
+    # salt in SLE is only available for SLE12 ASMM or SLES15 and variants of SLES but not SLED
+    if (is_jeos || is_opensuse || !is_staging && (check_var_array('SCC_ADDONS', 'asmm') || is_sle('15+') && !is_desktop)) {
```

Comment:
and I guess you can remove the `!is_staging` because there are no extra tests for staging

Reply:
One option could be to just remove the `check_var_array` for `asmm` and add that one to the `create_hdd_textmode` test.